packages/cli/src/commands/setup/jobs/jobsHandler.js

```diff
@@ -95,7 +95,7 @@ const tasks = async ({ force }) => {
             fs.mkdirSync(getPaths().api.jobs)
           } catch (e) {
             // ignore directory already existing
-            if (!e.message.match('file already exists')) {
+            if (!/file already exists/.test(e.message)) {
```

Comment:
> If you need to know if a string matches a regular expression [RegExp](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp), use [RegExp.prototype.test()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/test).

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/match#:~:text=Symbol.match%5D().-,If%20you%20need%20to%20know,-if%20a%20string

Reply:
Yeah I hate this syntax, it feels backwards to me. In real life I'd say "Is this car red?"  Not "Is the color red what this car is?" GROSS  If you start with a regex object (because it was passed in as an argument, or you constructed the regex from other variables) you use `test()`, but if you _start_ with the string you can use `match()`.

But if it's more performant or whatever, we can make it `test()` instead.

Reply:
Can't argue with that... I also think it reads backwards 🙁  But it is the recommended way!
*BUT* I changed the code to pass `{ recursive: true }`, which makes it so that no error is thrown if the directory already exists. So now we don't have to worry about any error messages 🎉